$OpenBSD: patch-configure_ac,v 1.5 2012/05/09 06:17:36 fgsch Exp $

Second part of this patch is to allow to compile libdvdnav with another
version installed.

--- configure.ac.orig	Fri Oct  7 21:20:59 2011
+++ configure.ac	Tue May  8 18:05:16 2012
@@ -143,6 +143,11 @@ case $host in
     THREAD_CFLAGS="-I/usr/local/include -D_THREAD_SAFE"
     CFLAGS="$THREAD_CFLAGS $CFLAGS"
     ;;
+  *-*-openbsd*)
+    THREAD_LIBS="-pthread"
+    THREAD_CFLAGS="-pthread"
+    CFLAGS="$CFLAGS $THREAD_CFLAGS"
+    ;;
   *mingw32* | *cygwin*)
     ;;
   *)
@@ -201,7 +206,7 @@ dnl ---------------------------------------------
 dnl cflags
 dnl ---------------------------------------------
 dnl Common cflags for all platforms
-CFLAGS="-O3 -Wall -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE $CFLAGS"
+CFLAGS="-Wall -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE $CFLAGS"
 DEBUG_CFLAGS="-g -DDEBUG $CFLAGS"
 
 AC_SUBST(DEBUG_CFLAGS)
@@ -214,7 +219,7 @@ AC_CHECK_PROG(DOXYGEN, doxygen, doxygen, no)
 dnl ---------------------------------------------
 dnl Some include paths ( !!! DO NOT REMOVE !!! )
 dnl ---------------------------------------------
-INCLUDES='-I$(top_srcdir) $(DVDNAV_CFLAGS)'
+INCLUDES='-I$(top_srcdir) -I$(top_srcdir)/src $(DVDNAV_CFLAGS)'
 AC_SUBST(INCLUDES)
 
 dnl ---------------------------------------------
